packages/webapp/src/views/common/legal-agreement-summary.html

```diff
@@ -0,0 +1,101 @@
+        'value': legalAgreementFileNames,
+        'valueDataTestId': 'legal-agreement-file-name-value',
+        'valueId': 'add-legal-agreement-file-value',
+        'href': '/land/check-you-added-all-legal-files',
+        'visuallyHiddenText':  ' legal agreement file',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':true
+    },
+    {
+        'text': 'Responsible body',
+        'value': responsibleBodies,
+        'valueDataTestId': 'legal-agreement-responsible-bodies-value',
+        'valueId': 'add-legal-agreement-responsible-bodies-value',
+        'href': '/land/check-responsible-bodies',
+        'visuallyHiddenText':  ' responsible body',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':true
+    },
+    {
+        'text': 'Landowner',
+        'value': landowners,
+        'valueDataTestId': 'legal-agreement-landowner-value',
+        'valueId': 'add-legal-agreement-landowner-value',
+        'href': '/land/check-landowners',
+        'visuallyHiddenText':  ' landowner',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':true
+    },
+    {
+        'text': 'Habitat management and monitoring plan included in legal agreement',
+        'value': habitatPlanSeperateDocumentYesNo,
+        'valueDataTestId': 'legal-agreement-habitatplan-type-value',
+        'valueId': 'add-legal-agreement-habitatplan-type-value',
+        'href': '/land/habitat-plan-legal-agreement',
+        'visuallyHiddenText':  ' habitat management plan',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':true
+    },
+    {
+        'text': 'Habitat management and monitoring plan uploaded',
+        'value': HabitatPlanFileName,
+        'valueDataTestId': 'habitat-plan-file-name-value',
+        'valueId': 'add-habitat-plan-file-value',
+        'href': '/land/check-habitat-plan-file',
+        'visuallyHiddenText':  ' habitat management plan file name',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':habitatPlanSeperateDocument
+    },
+    {
+        'text': 'Habitat enhancement works start date',
+        'value': HabitatWorksStartDate,
+        'valueDataTestId': 'habitat-works-start-date-value',
+        'valueId': 'add-habitat-works-start-date-value',
+        'href': '/land/enhancement-works-start-date',
+        'visuallyHiddenText':  ' enhancement works start date',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':true
+    },
+    {
+        'text': 'Legal agreement end date',
+        'value': HabitatWorksEndDate,
+        'valueDataTestId': 'habitat-works-end-date-value',
+        'valueId': 'add-habitat-works-end-date-value',
+        'href': '/land/legal-agreement-end-date',
+        'visuallyHiddenText':  ' legal agreement end date',
+        'classes': 'govuk-!-margin-bottom-7',
+        'show':true
+    },
+    {
+      'text': 'Local land charge search certificate file uploaded',
+      'value': localAndChargeFileName,

```

Comment:
Typo - local**L**AndChargeFileName. This looks like it would need correction throughout the code base.

Reply:
done